### DISCUSSION TITLE
Proxy Workflow

### DISCUSSION THREAD

**Discussion by leightv** [latenitefilms/BRAWToolbox#122] [Support]
Hi,
I have a client who I send a FCP library with Proxy files for them to rough cut and then they send me an XML file which I import into my main library which links to my optimised files.
Is there a recommended workflow to use BRAW files? Ideally I want to create proxy files which I send to my client in the library but the be able to link the clips back to the real BRAW files when I import the XML file.

Leigh

**Comment by latenitefilms** [comment]
Apologies for the delayed reply - I missed this one. I was having some issues with Gmail blocking some GitHub emails, however this is now fixed and I've added some other notification methods, so this shouldn't happen in the future.

As discussed in #124:

> Can you try create a new event, with a single `.mov` proxy in it. Make sure the `.braw` is in the same folder as the `.mov`.
> 
> Before dragging this event to the Toolbox, select the `.mov` in the Browser, then click **Clip > Open Clip**, and apply a Color Board effect to it (actually doesn't matter what the effect is, just an effect that does nothing) - then try again.

As discussed in #107, in a future BRAW Toolbox update, I'm hoping to add a feature that allows you to use QuickTime proxy files instead of BRAW files - but this will still require BRAW Toolbox. However, this should mean that you can import all the BRAW files on your machine, export proxies via Final Cut Pro's normal export function, and then on your client's machine, tell BRAW Toolbox to look for proxies in a certain location. I hope to have this functionality added sometime in May.
